int32 index
int32 pose
---
bool set_ok